README.md

```diff
@@ -1,2 +1,53 @@
-# Design-Teardown
-This project consists of building a heatmap of the Smashing magazine website
+# DESIGN TEARDOWN
+
+This project was developed as a learning project part of the Microverse curriculum.
+
+## Desktop
+![a gray-shade heatmap of the Smashing Magazine website.](./img/screenshot.PNG)
+
+## Built With
+
+- HTML5
+- CSS3
+- Google Fonts
+- Font awesome
+
+## Live Demo
+
+<a href= "https://rawcdn.githack.com/enkog/Design-Teardown/6d98b5c9306a1219663ea3424a5a56ef6fe86803/index.html" target="_blank">Live version</a>
```

Comment:
- [ ] Please, update your demo link, it is not displaying your latest commits.  As a recommendation, so you don't have to be manually updating this link during your code reviews, use the second link (development link) in Githack, that one will automatically load your last version in the same branch :)